modules/gocd/conf/cruise-config.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="utf-8"?>
+<cruise xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="cruise-config.xsd" schemaVersion="81">
+  <server artifactsdir="artifacts" agentAutoRegisterKey="123456789ytrewq" commandRepositoryLocation="default" serverId="e89d0b81-9c38-4e25-a946-ac1dc6944233" />
+  <pipelines group="MyApp">
+    <pipeline name="MyApp" template="Stakater_Template">
+      <params>
+        <param name="SOURCE_DIR" />
+        <param name="BUILD_DOCKER_IMAGE"></param>
+        <param name="BUILD_DOCKER_IMAGE_NAME"></param>
+        <param name="BUILD_DOCKER_IMAGE_OPTS"></param>
+        <param name="BUILT_WAR_FILE"></param>
+        <param name="APP_COMPILE_CMD"></param>
+        <param name="APP_TEST_CMD"></param>
+        <param name="APP_PACKAGE_CMD"></param>
+        <param name="APP_TEST_ARTIFACT">/</param>
+        <param name="APP_IMAGE_BUILD_DIR"></param>
+        <param name="APP_IMAGE_BUILD_VERSION">${GO_PIPELINE_COUNTER}.${GO_STAGE_COUNTER}</param>
+        <param name="APP_NAME">MyApplication</param>
```

Comment:
This is a sample name, the users are supposed to fill up all these params according to their needs